themes/bootstrap3/js/common.js

```diff
@@ -591,7 +591,12 @@ function setupAutocomplete() {
           ? event.detail
           : event.detail.value;
         input.value = value;
-        $("#searchForm").trigger("submit");
+        const $form = $searchbox.closest("form");
+        // Try to submit the form:
+        $form.trigger('submit');
+        // If form submit failed (which currently happens for course reserves),
+        // click the input button instead:
+        $form.find('input[type="submit"]').trigger('click');
```

Comment:
I think the issue here is that the form has multiple submit forms. We might be able to fix this issue by making each of the select+button pairs its own form. It's not invalid HTML but I imagine the different submits is what's holding up the JS.